src/model.rs

```diff
@@ -730,7 +730,7 @@ pub trait ModelWithProblem {
     fn cons_is_separated(&self, cons: &Constraint) -> bool;
 
     /// Writes the optimization model to a file with the given path and extension.
-    fn write(&self, path: &str, ext: &str) -> Result<(), Retcode>;
+    fn write(&self, path: &str, ext: &str, symb: bool) -> Result<(), Retcode>;
```

Comment:
This is the documentation that is the most important, as it is what the users would see first not the one in scip.rs. So please add the arguments description here too. Thank you!

Reply:
Ah ok, that's a good one. Since no other ones had the description I thought it was not necessary there. I'll apply the changes.